test/tests/ver-1fd/comparison_ver-1fd.py

```diff
@@ -0,0 +1,59 @@
+import csv
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+from scipy import special
+import os
+import math
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
+
+#===============================================================================
+# Extract TMAP8 results
+tmap8_sol = pd.read_csv("./gold/ver-1fd_out.csv")
+tmap8_sol_t = tmap8_sol['time']
+tmap8_sol_temperature = tmap8_sol['temperature_at_x']
+
+#===============================================================================
+# Calculate analytical solution
+def get_analytical_solution(x,t_vect):
+    """Returns the temperature as a function of the distance to the left side of the domain and time
+
+    Args:
+        x (float): distance in m
+        t (ndarray): time in s
+
+    Returns:
+        np.array: temperature in K at position x
+    """
+    T_initial = 100 # K
+    T_infinity = 500 # K
+    h = 200 # W/m^2/K)
+    k = 401 # W/m/K
+    rho_Cp = 3.439e6 # J/m^3/K
+    alpha = k/rho_Cp # ~1.17e-4 m^2/s
+    small_value = 1e-42 # to avoid division by 0
```

Comment:
Out of curiosity, is there any reason you used 1E-42 specifically?

Reply:
Since I'm picking a value that is very small to (1) prevent division by 0 and (2) not affect any subsequent times, I might as well use 42. 
I could probably have use 1e-50 or 1e-30, but it's not nearly as geeky.

Reply:
I am ashamed for not spotting that! Haha

Reply:
Nice :D